### PR TITLE
Ticket 25: header was added

### PR DESCRIPTION
**Pull request by maximFeldman1**
header was added to project

### CODE REVIEW

File: src/Components/App/AppRouter/index.tsx [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import { Suspense, lazy } from "react";
 import { Route, Routes, Navigate } from "react-router-dom";
 import { URL } from "../../../shared";
+import { Header } from "../../Layout/Header";
```
Comment (itairom-appsforce) [inline src/Components/App/AppRouter/index.tsx]: please use relative path

File: src/Components/Layout/Header/index.style.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
+
+const HeaderWraper = styled.div`
```
Comment (itairom-appsforce) [inline src/Components/Layout/Header/index.style.tsx]:
```suggestion
const HeaderStyle = styled.div`
```

Comment (itairom-appsforce) [inline src/Components/Layout/Header/index.style.tsx]: we should use primereact components, i will do overview

**Review by amitmarko — APPROVED** [review]
LGTM

File: src/App.tsx [hunk 3]
```diff
@@ -2,12 +2,16 @@ import React from "react";
...
 function App() {
   return (
-    <BrowserRouter>
-      <AppRouter />
-    </BrowserRouter>
+    <>
+      <Header />
+      <BrowserRouter>
+        <AppRouter />
+      </BrowserRouter>
```
Comment (amitmarko) [inline src/App.tsx:13]: Move BrowserRouter above the Header
```suggestion
      <BrowserRouter>
          <Header />
        <AppRouter />
      </BrowserRouter>
```

File: src/Components/Layout/Header/index.style.tsx (hunk 2 above)
Comment (maximFeldman1) [reply src/Components/Layout/Header/index.style.tsx]: Fixed

File: src/App.tsx (hunk 3 above)
Comment (maximFeldman1) [reply src/App.tsx:13]: Fixed

File: src/Components/Layout/Header/index.style.tsx (hunk 2 above)
Comment (maximFeldman1) [reply src/Components/Layout/Header/index.style.tsx]: i change to Root(Amit requirement)

Comment (maximFeldman1) [reply src/Components/Layout/Header/index.style.tsx]: for header i used style components (Amit requirement)

File: src/App.tsx [hunk 4]
```diff
@@ -2,12 +2,16 @@ import React from "react";
 import "./App.css";
 import { BrowserRouter } from "react-router-dom";
 import AppRouter from "./Components/App/AppRouter";
+import { Header } from "./Components/Layout/Header";
```
Comment (itairom-appsforce) [inline src/App.tsx:5]: change folder name to lowercase (also for page), we use Capitalize for component Folder (eg 'Header')

**Pull request merged**